guides/07_FAQs.md

```diff
@@ -95,7 +95,27 @@ In Firefox, if access to camera or microphone has been granted to the site — e
 If the page itself is loaded via a different network interface than the one that is intended to be used by the Amazon Chime SDK to connect to Amazon Chime media resources, _e.g._, in a split-tunneling VPN where browser traffic uses the VPN interface but Amazon Chime video and audio does not, then ICE gathering will use the wrong interface, which can result in sub-optimal network routing or an inability to use audio or video functionality.
 
 Customers and end users must ensure that either (a) end users do not use SDK applications in these kinds of split-tunneling scenarios, or (b) the SDK application always requests microphone permissions prior to beginning ICE.
- 
+
+## Known Build Issues
+
+### Why is my build with rollup failing after upgrading to Amazon Chime JS SDK 3.8.0?
```

Comment:
I think it started failing from 3.7.0 since messaging client sdk dependency was added in 3.7.0.

Reply:
Amazon Chime JS SDK -> Amazon Chime SDK for JavaScript. Think we do not follow it more generally but would be good to follow the convention :)

Reply:
Oh yeah that's correct. It is 3.7.0. That's a typo.